public/local/panel.html

```diff
@@ -14,5 +14,7 @@
         <h2>Hello viewer!</h2>
         <h3>Poll down here</h3>
         <div id="poll"></div>
+        <div id="choiceA"></div>
```

Comment:
this is fine when there are exactly two choices - even better would be to use jquery to programmatically add in the choices as buttons in some kind of multiple choice input